crates/router/src/core/payment_methods/transformers.rs

```diff
@@ -323,7 +323,7 @@ pub async fn mk_add_locker_request_hs<'a>(
     url.push_str("/cards/add");
     let mut request = services::Request::new(services::Method::Post, &url);
     request.add_header(headers::CONTENT_TYPE, "application/json".into());
-    request.set_body(RequestContent::Json(Box::new(body.to_string())));
+    request.set_body(RequestContent::Json(Box::new(body)));
```

Comment:
this isn't right, can you please use `jwe_payload` directly instead of using body. as body encodes the payload into `serde_json::Value`